drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -871,6 +871,7 @@ function DragAndDropBlock(runtime, element, configuration) {
                 input: input,
                 displayName: item.displayName,
                 imageURL: imageURL,
```

Comment:
I don't feel strongly about the variable names either, so I'll just leave it the way the current version happens to be.

The problem with the fallback was that `expandedImageURL` would be undefined if the old attribute name was used, but the JS code used it anyway in that case, so it was indeed a bug (fixed now).